Makefile

```diff
@@ -14,7 +14,6 @@ DESIGN_DIR=design
 DESIGNS := $(shell find $(SOURCE_DIR)/$(DESIGN_DIR) -path $(SOURCE_DIR)/vendor -prune -o -name '*.go' -print)
 
 GOLINT_DIRS=$(shell go list -f {{.Dir}} ./... | grep -v -E "vendor|app|client|tool/cli")
```

Comment:
In `.make/test.mk` we have defined `ALL_PKGS_EXCLUDE_PATTERN = 'vendor\|app\|tool\/cli\|design\|client\|test'`.  Notice that you didn't exclude all dirs. Maybe it makes sense to move your targets over to that file and use the variable above instead?